### PR TITLE
refactor(ekf_localizer): add Simple1DFilter params to parameter file

### PR DESCRIPTION
**Pull request by meliketanrikulu**
## Description

Simple1DFilter parameters will be added to the parameter file to be able to change these parameters from the parameter file.

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: localization/ekf_localizer/config/ekf_localizer.param.yaml [hunk 1]
```diff
@@ -22,6 +22,11 @@
     proc_stddev_vx_c: 10.0
     proc_stddev_wz_c: 5.0
 
+    #Simple1DFilter parameters
+    z_filter_proc_dev: 1.0
+    roll_filter_proc_dev: 0.01
+    pitch_filter_proc_dev: 0.01
+
```
Comment (kminoda) [inline localization/ekf_localizer/config/ekf_localizer.param.yaml:29]: Please add these parameters in Readme as well

**Review by kminoda — COMMENTED** [review]
Thank you for the PR! Please fix the semantic-pull-request

File: localization/ekf_localizer/config/ekf_localizer.param.yaml (hunk 1 above)
Comment (meliketanrikulu) [reply localization/ekf_localizer/config/ekf_localizer.param.yaml:29]: @kminoda . Could you check again

**Review by kminoda — APPROVED** [review]
LGTM.
(I renamed the title prefix from `feat` to `refactor`, but open to discussion if you have any concerns for it)

**Comment by meliketanrikulu** [comment]
> LGTM. (I renamed the title prefix from `feat` to `refactor`, but open to discussion if you have any concerns for it)

Thank you @kminoda . Could you check this PR too. it is related this PR :
 https://github.com/autowarefoundation/autoware_launch/pull/710

**Pull request merged**